setup.py

```diff
@@ -14,7 +14,7 @@
 # or pip install -e .
 install_requirements = [
     # Install squid-py and all its dependencies first
-    'common-utils-py==0.4.2',
+    'common-utils-py==0.4.3',
     'contracts-lib-py==0.5.2',
```

Comment:
Can we upgrade to 0.5.4 in contracts lib to make it work with rinkeby

Reply:
sure